main.go

```diff
@@ -131,6 +131,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	copyTaskController, err := controllers.NewEtcdEtcdCopyBackupsTaskReconcilerWithImageVector(mgr)
```

Comment:
```suggestion
	etcdCopyBackupsTaskController, err := controllers.NewEtcdEtcdCopyBackupsTaskReconcilerWithImageVector(mgr)
```
(or, `ecbtController`)